static/resources/json/datadog_collection.json

```diff
@@ -296,90 +1515,241 @@
+									"value": "gzip"
+								},
+								{
+									"key": "DD-POOL",
+									"value": "dogweb_sameorig"
+								},
+								{
+									"key": "X-Frame-Options",
+									"value": "SAMEORIGIN"
+								},
+								{
+									"key": "X-Content-Type-Options",
+									"value": "nosniff"
+								},
+								{
+									"key": "Strict-Transport-Security",
+									"value": "max-age=15724800;"
+								}
+							],
+							"cookie": [],
+							"body": "{\n    \"notify_list\": [],\n    \"description\": \"An updated dashboard with memory info.\",\n    \"template_variables\": [\n        {\n            \"default\": \"myhost\",\n            \"prefix\": \"host\",\n            \"name\": \"host1\"\n        }\n    ],\n    \"is_read_only\": true,\n    \"id\": \"civ-zbj-ag5\",\n    \"title\": \"<NEW_DASHBOARD_TITLE>\",\n    \"url\": \"/dashboard/civ-zbj-ag5/newdashboardtitle\",\n    \"created_at\": \"2019-04-19T12:35:20.490074+00:00\",\n    \"modified_at\": \"2019-04-19T12:36:11.667609+00:00\",\n    \"author_handle\": \"test@datadoghq.com\",\n    \"widgets\": [\n        {\n            \"definition\": {\n                \"requests\": [\n                    {\n                        \"q\": \"sum:system.mem.free{*}\"\n                    }\n                ],\n                \"type\": \"timeseries\",\n                \"title\": \"Sum Memory Free Shell\"\n            },\n            \"id\": 8077021146151090\n        }\n    ],\n    \"layout_type\": \"ordered\"\n}"
+						}
+					]
 				},
 				{
-					"name": "Update a Dashboard",
+					"name": "Get all Dashboards",
 					"request": {
-						"method": "PUT",
+						"method": "GET",
 						"header": [
 							{
 								"key": "Content-Type",
-								"name": "Content-Type",
-								"value": "application/json",
-								"type": "text"
+								"value": "application/json"
 							}
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n    \"title\": \"Sum of Memory Free 2\",\n    \"widgets\": [\n        {\n            \"definition\": {\n                \"type\": \"timeseries\",\n                \"requests\": [\n                    {\n                        \"q\": \"sum:system.mem.free{*}\"\n                    }\n                ],\n                \"title\": \"Sum Memory Free Shell\"\n            }\n        }\n    ],\n    \"layout_type\": \"ordered\",\n    \"description\": \"An updated dashboard with memory info.\",\n    \"is_read_only\": true,\n    \"notify_list\": [\n        \"test@datadoghq.com\"\n    ],\n    \"template_variables\": [\n        {\n            \"name\": \"host1\",\n            \"prefix\": \"host\",\n            \"default\": \"myhost\"\n        }\n    ]\n}"
+							"raw": ""
 						},
 						"url": {
-							"raw": "https://api.datadoghq.com/api/v1/dashboard/:DASHBOARD_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+							"raw": "https://app.datadoghq.com/api/v1/dashboard/?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```

Comment:
It's `app` here and `api` below. Looks like `api` is correct.

Reply:
it should be api indeed, updating the collection in another commit